app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java

```diff
@@ -145,11 +146,15 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 				infectedView.setVisibility(View.VISIBLE);
 
 				// Show the onset date of the report
-				long onsetDateInMillis = secureStorage.getPositiveReportOnsetDate();
-				if (onsetDateInMillis > 0L) {
+				long oldestSharedKeyDateInMillis =
+						Math.max(secureStorage.getPositiveReportOldestSharedKey(),
+								System.currentTimeMillis() - MAX_EXPOSURE_AGE_MILLIS);
+				if (oldestSharedKeyDateInMillis > 0L) {
```

Comment:
Given `oldestSharedKeyDateInMillis` is at least `System.currentTimeMillis() - MAX_EXPOSURE_AGE_MILLIS`, this is never <= 0